core/modules/system/system.admin.inc

```diff
@@ -680,7 +680,6 @@ function system_modules($form, $form_state = array()) {
     '#value' => t('Save configuration'),
   );
   $form['#action'] = url('admin/modules/list/confirm');
-
```

Comment:
looks like a line was removed from here, probably by mistake?

Reply:
would that make a difference?

Reply:
Code execution-wise, it would not matter @arjunk2004 ...usually for clarity (especially in long functions), we keep the return in a separate line + an empty line before it. It helps visually when going through code.